### PR TITLE
feat: How to add a third party module

### PR DESCRIPTION
**Pull request by longieirl**
- Append a new sample project showing the introduction of a third party module i.e. xml-js

### CODE REVIEW

**Review by docirl — APPROVED** [review]

**Comment by longieirl** [comment]
@ashleytung can you please review, thanks.

File: thirdpartylibrary/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Adding External Library to your Fiori UI application
+
+# Prerequisites
+
+- You’ll need an SAP Business Technology Platform (SAP BTP) account
+- You are subscribed to SAP Business Application Studio, follow this [tutorial](https://help.sap.com/products/SAP%20Business%20Application%20Studio/9d1db9835307451daa8c930fbd9ab264/6331319fd9ea4f0ea5331e21df329539.html) for more information
+- Sub-account destination `northwind` is available, please see attached `northind` configuration
+
+# Sample Project
+A [Fiori tools Freestyle UI application](./ztravelapp/README.md) with an external library `xml-js` enabled.
```
Comment (ashleytung) [inline thirdpartylibrary/README.md:10]: freestyle app should be referred to as `freestyle SAPUI5` app. Also, I suggest to make the sentence a bit more descriptive. Example: "[This freestyle SAPUI5 application](./ztravelapp/README.md) demonstrates how to add an external library `xml-js` to the project"

File: thirdpartylibrary/ztravelapp/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Adding External Library to your Fiori UI application
```
Comment (ashleytung) [inline thirdpartylibrary/ztravelapp/README.md:1]: SAP Fiori instead of Fiori

File: thirdpartylibrary/ztravelapp/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+- You’ll need an SAP Business Technology Platform (SAP BTP) account
+- You are subscribed to SAP Business Application Studio, follow this [tutorial](https://help.sap.com/products/SAP%20Business%20Application%20Studio/9d1db9835307451daa8c930fbd9ab264/6331319fd9ea4f0ea5331e21df329539.html) for more information
+- Sub-account destination `northwind` is available
+
+# Create SAP Fiori Dev Space
+
+From your SAP BTP cockpit, select Instances and Subscriptions, select SAP Business Application Studio which will open a new tab into your dev space manager. Generate a SAP Fiori dev space.
+
+# Generate a Fiori UI Freestyle Application
```
Comment (ashleytung) [inline thirdpartylibrary/ztravelapp/README.md:15]: "freestyle SAPUI5 application"

**Review by ashleytung — CHANGES_REQUESTED** [review]
Hi John, i reviewed the read me files and added a a few comments on branding names.

File: thirdpartylibrary/ztravelapp/README.md (hunk 2 above)
Comment (longieirl) [reply thirdpartylibrary/ztravelapp/README.md:1]: Done

File: thirdpartylibrary/ztravelapp/README.md (hunk 3 above)
Comment (longieirl) [reply thirdpartylibrary/ztravelapp/README.md:15]: Done

**Review by ashleytung — APPROVED** [review]
looks good. Thank you.

**Comment by longieirl** [comment]
> looks good. Thank you.

Can you please merge as I dont have access? Thanks.

**Pull request merged**